src/frontend/shortlist/src/components/preferences/PreferenceDemo.vue

```diff
@@ -44,6 +47,36 @@ export default {
   },
   setup() {
     return {
+      question1: {
+        Name: "Instruction",
+        Text: "How much do you value instructions from your teachers?",
+        Min: 1,
+        MinLabel: "I'd like more time studying alone.",
+        Max: 7,
+        MaxLabel: "I wish instructors can cover every details",
```

Comment:
```suggestion
        MaxLabel: "More Important",
```

Reply:
"More class material coverage" 
how about this